src/meta/src/error.rs

```diff
@@ -132,6 +133,13 @@ pub enum MetaErrorInner {
 
     #[error("{0} has been deprecated, please use {1} instead.")]
     Deprecated(String, String),
+
+    #[error("Secret error: {0}")]
+    SecretError(
```

Comment:
What's this for?

Reply:
We will validate the connection on meta, which needs to fill in the secrets. So introducing `Secret Error` here for handling possible failure.